platforms/nuttx/NuttX/CMakeLists.txt

```diff
@@ -157,6 +159,15 @@ function(add_nuttx_dir nuttx_lib nuttx_lib_dir kernel extra target)
 		set(nuttx_lib_target lib${target}.a)
 	endif()
 
+	# Facilitate building kxx, as NuttX kernel should not contain CXX code
+	# and thus does not support creating libkxx natively...
+	# # This is a temporary _HACK_ and should be removed ASAP
```

Comment:
I don't think that it needs to be removed ASAP 🙂 Only after there is an alternative